cmd/client.go

```diff
@@ -65,6 +60,10 @@ var ClientCmd = &cli.Command{
 				if err != nil {
 					return fmt.Errorf("reading blob file: %w", err)
 				}
+				spaceDid, err := did.Parse(cCtx.String("space-did"))
```

Comment:
reason for move? just curious

Reply:
Purely accidental (was basing this on #52), will revert.